test/org/opensolaris/opengrok/configuration/RuntimeEnvironmentTest.java

```diff
@@ -298,19 +299,21 @@ public void testIgnoredNames() {
     @Test
     public void testUserPage() {
         RuntimeEnvironment instance = RuntimeEnvironment.getInstance();
-        String page = "http://www.myserver.org/viewProfile.jspa?username=";
-        assertEquals(page, instance.getUserPage());
-        instance.setUserPage(page.substring(5));
-        assertEquals(page.substring(5), instance.getUserPage());
```

Comment:
this should work. why is this change necessary ?

Reply:
I mean - the test should call setBugPage() at least once and verify that the setter works with getBugPage(). It **may** verify the default value as well. The more coverage we get the better.

Reply:
If you are referring to lines 142, this fails on a Windows box. The canonical path returned is C:\opengrok_automatic_test\foo\bar. The problem I was encountering has to do with the getCanonicalPath method in RuntemeEnvironment.java. It tests for existence, and when that fails, it returns the original string. So I was forced to make the original string be the canonical path to prevent the test failing further down.

So do you want to see something like this for the testBugPage:

        assertNull(instance.getBugPage());
        instance.setBugPage(instance.getBugPage());
        assertNull(instance.getBugPage());

Reply:
No, I am referring to the testUserPage(), testBugPage(), testReviewPage(). These tests should do something like this:

1. use getter to verify the default value is `null`
2. use setter to set a value
3. use getter to verify that the value was set